configure.ac

```diff
@@ -320,6 +319,46 @@ AS_CASE([$ENABLED_FIPS],
         AC_MSG_ERROR([Invalid value for --enable-fips "$ENABLED_FIPS" (main options: v1, v2, v5, ready, dev, rand, no, disabled)])
     ])
 
+# wolfEngine Options
+# This requires a FIPS source code bundle.
+AC_ARG_ENABLE([engine],
```

Comment:
I would move this above the `AS_CASE([$ENABLED_FIPS] ...`. Then, just set `ENABLED_FIPS` appropriately and that FIPS case logic will handle the FIPS version and whatnot. Less code repetition.